core/src/odbc_uri.rs

```diff
@@ -257,7 +257,7 @@ impl ODBCUri {
         Ok(())
     }
 
-    fn handle_uri(&mut self, uri: &str) -> Result<ClientOptions> {
+    fn handle_uri(&mut self, uri: &str) -> Result<(ClientOptions, Option<ConnectionString>)> {
```

Comment:
Why not either creating a struct for our option which encapsulates the Rust driver clientOption and the uuid represenrtation or return a pair like you are doing but simply returning an Option<UuidRepresentation> directly?
The reason why I'd like it better this way is that connection information logic will not be handled in 2 different places. Right now, it's Ok, but since you are passing the ConnectionString, I can see it drifting very easily and then make maintenance a little less easy

Reply:
Good question. I had thought about combining them into a unified return, but since they are treated differently by the Rust driver itself, I decided to keep them separate.